composer.json

```diff
@@ -1,63 +1,75 @@
 {
 	"name": "automattic/vipwpcs",
-	"type": "phpcodesniffer-standard",
 	"description": "PHP_CodeSniffer rules (sniffs) to enforce WordPress VIP minimum coding conventions",
+	"license": "MIT",
+	"type": "phpcodesniffer-standard",
 	"keywords": [
 		"phpcs",
 		"static analysis",
 		"standards",
 		"WordPress"
 	],
-	"license": "MIT",
 	"authors": [
 		{
 			"name": "Contributors",
 			"homepage": "https://github.com/Automattic/VIP-Coding-Standards/graphs/contributors"
 		}
 	],
+	"support": {
+		"issues": "https://github.com/Automattic/VIP-Coding-Standards/issues",
+		"wiki": "https://github.com/Automattic/VIP-Coding-Standards/wiki",
+		"source": "https://github.com/Automattic/VIP-Coding-Standards"
+	},
 	"require": {
 		"php": ">=5.4",
 		"dealerdirect/phpcodesniffer-composer-installer": "^0.4.1 || ^0.5 || ^0.6.2 || ^0.7",
 		"sirbrillig/phpcs-variable-analysis": "^2.11.1",
-		"squizlabs/php_codesniffer": "^3.5.5",
-		"wp-coding-standards/wpcs": "^2.3"
+		"squizlabs/php_codesniffer": "^3.7.1",
+		"wp-coding-standards/wpcs": "dev-develop"
 	},
 	"require-dev": {
-		"php-parallel-lint/php-parallel-lint": "^1.3.2",
 		"php-parallel-lint/php-console-highlighter": "^1.0.0",
+		"php-parallel-lint/php-parallel-lint": "^1.3.2",
 		"phpcompatibility/php-compatibility": "^9",
 		"phpcsstandards/phpcsdevtools": "^1.0",
 		"phpunit/phpunit": "^4 || ^5 || ^6 || ^7"
 	},
+	"minimum-stability": "dev",
+	"prefer-stable": true,
 	"config": {
 		"allow-plugins": {
 			"dealerdirect/phpcodesniffer-composer-installer": true
 		}
 	},
 	"scripts": {
-		"install-codestandards": "Dealerdirect\\Composer\\Plugin\\Installers\\PHPCodeSniffer\\Plugin::run",
```

Comment:
Just mentioning for the record: this commit also removed the `install-codestandards` script, which should no longer be needed. ✅